packages/coinstac-client-core/src/sub-api/project-service.js

```diff
@@ -33,19 +34,40 @@ class ProjectService extends ModelService {
     this.listeners = new Map();
   }
 
+  /**
+   * Get a metadata CSV's contents.
+   *
+   * @param {string} filename Full file path to CSV
+   * @returns {Promise<Project>}
+   */
   getCSV(filename) {
     return bluebird.promisify(fs.readFile)(filename)
       .then(data => bluebird.promisify(csvParse)(data.toString()))
       .then(JSON.stringify);
   }
 
   /**
+   * Load a metadata CSV file.
+   *
+   * @param {string} metaFilePath Path to metadata CSV
+   * @param {Array[]} metaFile Metadata CSV's contents
+   * @returns {File[]} Collection of files
+   */
+  getFilesFromMetadata(metaFilePath, metaFile) {
```

Comment:
Pulled this functionality from the UI’s _app/render/components/projects/form-project-controller.js_ component to re-use in _coinstac-simulator_.